<launch>
	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find grid_map_visualization)/launch/custom_rosconsole.conf"/>
 	<node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
 		 <param name="grid_map_topic" value="/grid_map" />
 		 
 		 <param name="point_cloud/point_type" value="type" />
 		 
 		 <param name="map_region/line_width" value="0.003" />
 		 <param name="map_region/color" value="16777215" />
 		 
 		 <param name="vector/type_prefix" value="vector" />
 		 <param name="vector/position_type" value="position" />
 		 <param name="vector/scale" value="1.0" />
 		 <param name="vector/line_width" value="0.001" />
 		 <param name="vector/color" value="16777215" />
 		 
 		 <param name="occupancy_grid/type" value="occupancy" />
 		 <param name="occupancy_grid/data_min" value="0.0" />
 		 <param name="occupancy_grid/data_max" value="1.0" />
    </node>
</launch>
